stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessagesViewModelFactory.kt

```diff
@@ -80,7 +83,8 @@ public class MessagesViewModelFactory(
                 ClipboardHandlerImpl(context.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager),
                 showDateSeparators = showDateSeparators,
                 showSystemMessages = showSystemMessages,
-                deletedMessageVisibility = deletedMessageVisibility
+                deletedMessageVisibility = deletedMessageVisibility,
+                messageFooterVisibility = messageFooterVisibility
```

Comment:
Not related to this PR, but seems that `dateSeparatorThresholdMillis` param is missing here.

Reply:
Good point, will add!